tools/links/links.config.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+$usageFiles = call_user_func(function (array $a): array {
+    asort($a);
+    return $a;
+}, (new Finder('./docs'))
+    ->includeName('*.md')
+    ->excludeWholeName('./docs/release_notes/*')//TMP
+    ->excludeWholeName('./docs/snippets/*')//TMP
+    ->find());
+
+$resourceFiles = call_user_func(function (array $a): array {
+    asort($a);
+    return $a;
+},
+    // Images
+    (new Finder('./docs'))
+        ->includeName('*.png')
+        ->includeName('*.jpg')
+        ->find(),
+);
+
+$exclusionTests = array_merge_recursive(UrlTester::getDefaultExclusionTests(), [
+    'url' => [
+        function (string $url, string $file = null): bool {
+            return 'docs/index.md' === $file && str_starts_with($url, 'docs/');
+        },
+        function (string $url, string $file = null): bool {
+            // ibexa.co APIs needing authentication, namespaces, commercial aliases, etc.
+            return str_starts_with($url, 'https://updates.ibexa.co') // 401
+                || str_starts_with($url, 'https://support.ibexa.co') // 302 → /login
+                || str_starts_with($url, 'https://connect.ibexa.co') // 302 → https://ibexa.integromat.celonis.com/
+                || str_starts_with($url, 'http://ibexa.co/namespaces/') // 301
+                || str_starts_with($url, 'http://ibexa.co/xmlns/') // 301
+                || str_starts_with($url, 'http://ez.no/namespaces/') //301
+                || str_starts_with($url, 'https://api.cloud.ibexa.co') // 301, PLATFORMSH_CLI_API_URL
+                //|| str_starts_with($url, 'https://admin.perso.ibexa.co/api/') // 400
+                || str_starts_with($url, 'https://admin.perso.ibexa.co/') // 404
+                || str_starts_with($url, 'https://event.perso.ibexa.co/api/') // 400
```

Comment:
```suggestion
                || str_starts_with($url, 'https://event.perso.ibexa.co/api/') // 404
                || str_starts_with($url, 'https://tracker.ibexa.co/api/') // Could not resolve host
```

Reply:
Check this subdomain introduced in https://github.com/ibexa/documentation-developer/commit/363844251565692d4e16225242a0a52c0ef3fca0#diff-38a9c21114f72dda086ab70f5ed479a451a285a263de16e8a8a4867a4996334cL450-R511